src/framework/components/camera/component.js

```diff
@@ -134,45 +134,26 @@ Object.assign(pc, function () {
     });
 
     /**
-     * @name pc.CameraComponent#vrDisplay
-     * @type {pc.VrDisplay}
-     * @description The {@link pc.VrDisplay} that the camera is current displaying to. This is set automatically by calls to {@link pc.CameraComponent#enterVr}
-     * or {@link pc.CameraComponent#exitVr}. Setting this property to a display directly enables the camera to use the transformation information
-     * from a display without rendering stereo to it, e.g. for "magic window" style experiences.
-     * @example
-     * // enable magic window style interface
-     * var display = this.app.vr.display;
-     * if (display) {
-     *     this.entity.camera.vrDisplay = display;
-     * }
-     *
-     * var camera = this.entity.camera;
-     * camera.enterVr(function (err) {
-     *     if (err) return;
-     *     var display = camera.vrDisplay; // access presenting pc.VrDisplay
-     * });
+     * @readonly
+     * @name pc.CameraComponent#node
+     * @type pc.GraphNode
+     * @description Queries the camera's GraphNode. Can be used to get position and rotation.
      */
-    Object.defineProperty(CameraComponent.prototype, "vrDisplay", {
+    Object.defineProperty(CameraComponent.prototype, "node", {
         get: function () {
-            return this.data.camera.vrDisplay;
-        },
-        set: function (value) {
-            this.data.camera.vrDisplay = value;
-            if (value) {
-                value._camera = this.data.camera;
-            }
+            return this.data.camera._node;
         }
     });
 
     /**
      * @readonly
-     * @name pc.CameraComponent#node
-     * @type {pc.GraphNode}
-     * @description Queries the camera's GraphNode. Can be used to get position and rotation.
+     * @name pc.CameraComponent#xr

```

Comment:
Do you think `pc.CameraComponent#xr` will be confused with `pc.Application#xr`? In other words, might the developer assume it's another `pc.XrManager` instance? Alternative naming suggestions: `xrActive`, `xrEnabled`.

Reply:
Based on other naming conventions, do you think `isXr` is reasonable naming for this property?

Reply:
Just chatted with the team. The most preferred naming is `pc.CameraComponent#xrEnabled`. Let's go with that. 😄

Reply:
I actually couldn't think of use case for this property, so removed it entirely.